addons/overpressure/XEH_postInit.sqf

```diff
@@ -1,9 +1,13 @@
 #include "script_component.hpp"
 
 ["CBA_settingsInitialized", {
-    TRACE_1("settingsInit eh",GVAR(distanceCoefficient));
-    if (GVAR(distanceCoefficient) <= 0) exitWith {};
 
+    TRACE_1("settingsInit eh",GVAR(backblastDistanceCoefficient));
+    TRACE_1("settingsInit eh",GVAR(overpressureDistanceCoefficient));
+
+    // Both settings disabled, no need to initialize overpressure.
+    if (GVAR(backblastDistanceCoefficient) <= 0 && GVAR(overpressureDistanceCoefficient) <= 0) exitWith {};
```

Comment:
Requires both settings having the lower bound set to 0 instead of -1.
```suggestion
    TRACE_2("settingsInit eh",GVAR(backblastDistanceCoefficient),GVAR(overpressureDistanceCoefficient));

    // Both settings disabled, no need to initialize overpressure.
    if (GVAR(backblastDistanceCoefficient) == 0 && GVAR(overpressureDistanceCoefficient) == 0) exitWith {};
```